src/NServiceBus.Core/Reliability/Outbox/TransportOperation.cs

```diff
@@ -33,7 +34,7 @@ public TransportOperation(string messageId, Dictionary<string, string> options,
         /// <summary>
         /// Sending options.
         /// </summary>
-        public Dictionary<string, string> Options { get; }
+        public OperationProperties Options { get; }
```

Comment:
Should this be named `Properties` instead of `Options`? They are called `Properties` in the `TransportOperation` class in the `Transports` namespace.

Reply:
makes sense, yes